src/test/java/org/xbib/elasticsearch/index/mapper/langdetect/DetectLanguageTest.java

```diff
@@ -34,14 +55,271 @@ public void testKorean() throws IOException {
         testLanguage("korean.txt", "ko");
     }
 
+    @Test
+    public void testUdhrAccuraciesDefaultProfileDefaultLanguages() throws IOException {
```

Comment:
These tests are a bit repetitive, but I'm not sure whether looping over these parameters in a loop would be much cleaner/clearer.